akka-docs/src/test/scala/docs/stream/operators/sourceorflow/Throttle.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.ThrottleMode
+import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.Source
+
+import scala.concurrent.duration._
+
+/**
+ *
+ */
+object Throttle extends App {
+
+  implicit val sys = ActorSystem("25fps-stream")
+
+  val frameSource: Source[Int, NotUsed] =
+    Source.fromIterator(() => Iterator.from(0))
+
+  // #throttle
+  val framesPerSecond = 24
+
+  // val frameSource: Source[Frame,_]
+  val videoThrottling = frameSource.throttle(
+    framesPerSecond,
+    1.second,
+    framesPerSecond * 30, // maximumBurst
+    ThrottleMode.shaping)
```

Comment:
Well done to find a real use case for that maximumBurst parameter. Anyway I wonder if we should start with an example that is not using it, (only elements / 1.second)? I think that is much more commonly used and I think it's rather easy to [misconfigure the burst stuff](https://github.com/akka/akka/pull/24784).

Could be same example without the 30 sec burst, and then expand it with that addition?

Reply:
Sure! 👍🏼

Reply:
The split example looks better. :-)